internal/pkg/externalplugins/blunderbuss/blunderbuss.go

```diff
@@ -159,6 +194,11 @@ func HandleIssueCommentEvent(gc githubClient, ce *github.IssueCommentEvent, cfg
 
 func handle(ghc githubClient, opts *externalplugins.TiCommunityBlunderbuss, repo *github.Repo, pr *github.PullRequest,
 	log *logrus.Entry, ol ownersclient.OwnersLoader) error {
+	if opts.RequireSigLabel && !isPRContainSigLabel(pr) {
```

Comment:
I think we need move this check into `HandlePullRequestEvent`.

In addition, there is no need to report errors, just a log entry.

Reply:
> In addition, there is no need to report errors, just a log entry.

Do you mean to use `log.Errorf()` instead of returning an error?

Reply:
> I think we need move this check into `HandlePullRequestEvent`.

Do we need to check for the issue_comment event?

Reply:
> In addition, there is no need to report errors, just a log entry.

Do you mean to use `log.Errorf()` instead of returning an error?

Reply:
> I think we need move this check into `HandlePullRequestEvent`.

Do we need to check for the issue_comment event?

Reply:
Yes, also check for the issue_comment.

Reply:
> Do you mean to use `log.Errorf()` instead of returning an error?


log.Info because this is a feature not an exception.